b2sdk/b2http.py

```diff
@@ -235,34 +244,77 @@ def post_content_return_json(
            except B2Error as e:
                ...
 
-        :param str url: a URL to call
-        :param dict headers: headers to send.
-        :param data: bytes (Python 3) or str (Python 2), or a file-like object, to send
-        :return: a dict that is the decoded JSON
-        :rtype: dict
+        :param method: HTTP method: 'POST' or 'GET'
+        :param url: a URL to call
+        :param headers: headers to send.
+        :param data: raw bytes or a file-like object to send
+        :param try_count: a number of retries
+        :param params: a dict that will be converted to query string for GET requests or additional metadata for POST requests
```

Comment:
"Additional metadata" is confusing, in fact it will be simply dropped for non-GET requests

Reply:
for POST request it is still used in error-translation function